### DISCUSSION TITLE
About import remote libs

### DISCUSSION THREAD

**Discussion by libaice** [ProjectOpenSea/seadrop#14] [Q&A] [answered]
Hi, I have a question about the others repo libs for this contract . 
when I had downloaded the seaport , and then use `yarn` to install all dependecies , and then found that all other libs cannot be downloaded . hoow to slove this problem .

need I download all this repos mannually to libs folder ?

**Comment by esren0x** [comment, marked as answer]
> Hi, I have a question about the others repo libs for this contract . when I had downloaded the seaport , and then use `yarn` to install all dependecies , and then found that all other libs cannot be downloaded . hoow to slove this problem .
> 
> need I download all this repos mannually to libs folder ?

You need to tell git that you want to clone the submodules too
` git clone --recursive https://github.com/ProjectOpenSea/seadrop.git`

**Reply by akshitv3** [reply]
issue still persists 

Cloning into '/home/akshit/seadrop/lib/ERC721A-Upgradeable'...
git@github.com: Permission denied (publickey).
fatal: Could not read from remote repository.

Please make sure you have the correct access rights
and the repository exists.
fatal: clone of 'git@github.com:chiru-labs/ERC721A-Upgradeable.git' into submodule path '/home/akshit/seadrop/lib/ERC721A-Upgradeable' failed
Failed to clone 'lib/ERC721A-Upgradeable' a second time, aborting
